docs/guides/chia-gaming/gaming-california-poker-rules.md

```diff
@@ -0,0 +1,61 @@
+---
+slug: /guides/gaming-california-poker-rules
+title: California Poker Rules
+---
+
+## Objective
+
+California Poker is a card game where players exchange cards and compete to make the best five-card poker hand. Each player receives 8 cards, selects 4 to keep, and swaps the other 4 to their opponent. The player who can make the best five-card poker hand from their 8 cards after swapping wins.
```

Comment:
Maybe this is just semantics, but doesn't the player select the 4 cards to **exchange**?

In addition, you might want to state that both players can see the other players' cards. It's a unique facet of this game compared to similar poker games.

Reply:
thank you for catching this! I caught my other typo of this but missed this instance